src/gameHandlerClass.ts

```diff
@@ -35,4 +35,15 @@ export class GameHandler {
       ? this._playerTwo
       : this._playerOne;
   }
+
+  canIUseMango(): Boolean {
+    return this.getCurrentPlayer().hasMango;
+  }
+
+  useMango(): void | Error {
+    if (this.canIUseMango() == false)
```

Comment:
I meant "canUseMango" inside of player class, not in gameHandlerClass :)
it is responsibility of player, not of gameHandler

Reply:
`this.canIUseMango() == false` it's the same as:
`!this.canIUseMango()`